DigitalEcosystem/utils/fingerprints.py

```diff
@@ -73,3 +82,50 @@ def fingerprint_ewald_sum(max_atoms: int, df: pd.DataFrame,) -> np.ndarray:
     result = np.real(np.vstack(df['atoms_object (unitless)'].apply(ewald_matrix.create)))
 
     return result
+
+
+# ============
+# Descriptors
+# ============
```

Comment:
We need to explain the naming above - what is Fingerprint? Descriptor?

Reply:
Added comments to the comments